tools/respecDocWriter.js

```diff
@@ -292,17 +289,13 @@ function handleConsoleMessages(page, haltFlags, onError, onWarning) {
     }
     switch (type) {
       case "error":
-        onError({ message: text });
-        haltFlags.error = true;
-        break;
+        return onError({ message: text });
       case "warning":
         // Ignore polling of respecDone
         if (/document\.respecDone/.test(text)) {
           return;
         }
```

Comment:
I think we can remove this now? Or is it needed in some really old ReSpec?
```suggestion
```

Reply:
Yeah, we can remove.